docs/source/developing/writing-docs.adoc

```diff
@@ -17,6 +17,9 @@ documentation is a mix of generated
 https://en.wikipedia.org/wiki/Markdown[Markdown] files and manually
 maintained https://en.wikipedia.org/wiki/AsciiDoc[Asciidoc] files.
 
+If you are adding a new file to the doumentation please update
+ the `_topic_map.yml`  present in the root of `docs/source` sub-folder.
```

Comment:
If you add a new file to the documentation...
you can remove the word 'present'